### ISSUE TITLE
why evaporationAvgTime parameter has an impact on land use display

### ISSUE THREAD

**Issue by benoitgaudou** [CityScope/CSL_Hanoi#22]
Looking at the landuse species code, we can see:
```
species landuse schedules:[]{
	string type;
	rgb color;
	aspect base{
		if(evaporationAvgTime>2000){
		  draw shape color:color border:#black;	
		}	
	}
}
```

I do not get the reason why the parameter related to evaporation speed also controls the display of the landuse.
In particular, from a user point of view, it is quite strange to observe.

**Comment by agrignard** [comment]
The goal is here is to give an easy way for the user to trigger two views using the slider. We can discuss about this but I think it's good to be able to disable the landuse to really focus on the water.

Again the slider is really the cherry on the cake, if we want to have it we need to think also about the bow (made in lego here) that will embed the slider

**Comment by benoitgaudou** [comment]
Perhaps it could be time to design what you precisely want as an additional remote control for the simulation, to see if we can do something ......

**Comment by tnguyenh** [comment]
It sound strange that one slider controls two different things...

**Comment by agrignard** [comment]
I insist that this slider is NOT PART of the initial design it was here for another table (urbam) so what do you mean by additional remote control?

**Comment by agrignard** [comment]
OK so now I removed (as it seems that it was confusing) the double impact. So now there is no way to remove the landuse display and the slider is controlling the evaporation rate (evaporationAvgTime)

I personally think that the slider is maybe not that crucial. 

We can discuss togetehr to know which parameter it should trigger if we have one.

I close this issue as the initial problem has been removed
